File: apps/mobile/ios/NetTapu/LiquidGlassViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(LiquidGlassViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(cornerRadius, CGFloat)
RCT_EXPORT_VIEW_PROPERTY(activeIndex, NSInteger)
RCT_EXPORT_VIEW_PROPERTY(tabCount, NSInteger)

@end

File: apps/mobile/ios/NetTapu/LiquidGlassViewManager.swift
import Foundation
import React

/// React Native bridge — exposes LiquidGlassView as <LiquidGlassView> in JS
@objc(LiquidGlassViewManager)
class LiquidGlassViewManager: RCTViewManager {

  override func view() -> UIView! {
    return LiquidGlassView()
  }

  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
